lib/index.ts

```diff
@@ -126,17 +133,41 @@ const sdk = (...args: Parameters<typeof createSdk>) => {
       }
 
       try {
-        const token = await this.validateToken(sessionJwt)
-        return { ...token, sessionJwt }
+        const token = await sdk.validateJwt(sessionJwt)
+        return token
       } catch (error) {
-        logger?.error('failed to validate session token from access key', error)
+        logger?.error('failed to parse jwt from access key', error)
         throw Error('could not exchange access key')
       }
     },
+
+    validatePermissions(authInfo: AuthenticationInfo, permissions: string[]): boolean {
+      return sdk.validateTenantPermissions(authInfo, null, permissions)
+    },
+
+    validateTenantPermissions(
+      authInfo: AuthenticationInfo,
+      tenant: string,
+      permissions: string[],
+    ): boolean {
+      const granted = getAuthorizationClaimItems(authInfo, permissionsClaimName, tenant)
+      return permissions.every((perm) => granted.includes(perm))
+    },
+
+    validateRoles(authInfo: AuthenticationInfo, roles: string[]): boolean {
+      return sdk.validateTenantRoles(authInfo, null, roles)
+    },
+
+    validateTenantRoles(authInfo: AuthenticationInfo, tenant: string, roles: string[]): boolean {
+      const membership = getAuthorizationClaimItems(authInfo, rolesClaimName, tenant)
+      return roles.every((role) => membership.includes(role))
+    },
```

Comment:
Isn't it exactly the same as `validateTenantPermissions` & ` validatePermissions`?